iwyu_globals.cc

```diff
@@ -179,8 +183,10 @@ int CommandlineFlags::ParseArgv(int argc, char** argv) {
     {"max_line_length", optional_argument, nullptr, 'l'},
     {"no_comments", optional_argument, nullptr, 'o'},
     {"no_fwd_decls", optional_argument, nullptr, 'f'},
+    {"tolerate_transitive", optional_argument, nullptr, 'r'},
     {nullptr, 0, nullptr, 0}
   };
+  // TODO: This doesn't cover all options!
```

Comment:
I think that's intentional, not all options are exposed as short-opts.

Reply:
I'll remove it then